gwswlib/hydx.py

```diff
@@ -8,6 +8,47 @@ class Hydx:
     def __init__(self):
         self.connection_nodes = []
 
+    def check_import_data(self):
+        self._check_on_unique(
+            self.connection_nodes, "identificatieknooppuntofverbinding", True, "knoop"
+        )
+
+    def _check_on_unique(
+        self,
+        records,
+        unique_field,
+        remove_doubles=False,
+        item_name_for_logging="",
+        log_level=logging.WARNING,
```

Comment:
Log level is een rare hier: dat stel je meestal niet in. Een `-v` om meer verbose logging te krijgen is meestal voldoende, toch?
(En volgens mij gebruik je hem ook niet).